mappings/net/minecraft/entity/ai/brain/WalkTarget.mapping

```diff
@@ -1,9 +1,9 @@
 CLASS aot net/minecraft/entity/ai/brain/WalkTarget
 	FIELD a lookTarget Lakg;
-	FIELD b speedFactor F
+	FIELD b speed F
```

Comment:
Why this change?

Reply:
It's eventually passed to `startMovingAlong(@Nullable final Path path, final double speed)` in `EntityNavigation`, which calls it "speed", not "speedFactor".